src/qt/forms/debugwindow.ui

```diff
@@ -984,6 +984,117 @@
            </size>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_8">
+            <item>
+              <layout class="QHBoxLayout" name="horizontalLayout_8">
+                <property name="topMargin">
+                  <number>0</number>
+                </property>
+                <property name="bottomMargin">
+                  <number>0</number>
+                </property>
+                <item>
+                  <widget class="QLabel" name="peerHeading">
```

Comment:
Looks like this structure is duplicated, is this correct?

Reply:
thanks - taking a look right now.

Reply:
thank you!